force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,10 +212,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
-    }    
+        Integer MILLISECONDS_PER_MINUTE = 60000; // Make this value a constant
```

Comment:
You are missing the keyword here to make this a constant. There is an example in one the problems below.